include/circt/Dialect/MSFT/MSFTAttributes.td

```diff
@@ -61,3 +61,30 @@ def PhysLocation : MSFT_Attr<"PhysLocation"> {
     "PrimitiveTypeAttr":$primitiveType,
     "uint64_t":$x, "uint64_t":$y, "uint64_t":$num);
 }
+
+def PhysicalBounds : MSFT_Attr<"PhysicalBounds"> {
+  let summary = "Describes a rectangle bounding a physical region on a device";
```

Comment:
Should probably specify whether or not it's inclusive in a description.

Reply:
Good point... I will look up which it is an mention that.